rabbit-escape-engine/src/rabbitescape/engine/solution/SolutionCommand.java

```diff
@@ -12,7 +12,14 @@
 
     public SolutionCommand( CommandAction... actions )
     {
-        this.actions = actions;
+        if ( 0 == actions.length )
+        {
+            this.actions = new CommandAction[] { new WaitAction( 1 ) };
+        }
+        else
+        {
+            this.actions = actions;
+        }
     }
```

Comment:
I did something like that at first. But null actions don't exist, for example ';;;;' has implied ones like '1;1;1;1;', so this felt right. I can do it the other way if you insist.